### PR TITLE
Added Ordinal 211 filelength & test for it

### PR DESCRIPTION
**Pull request by tuday2**

### CODE REVIEW

**Comment by tuday2** [comment]
Closes #181

File: MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs [hunk 1]
```diff
@@ -7699,5 +7703,28 @@ private void stp4cs()
...
+        ///     filelength - gets file size in bytes
+        ///
+        ///     Signature: long filelength(int handle);
+        /// </summary>
+        private void filelength()
+        {
+            var fileHandle = GetParameter(0);
+            
+            if (FilePointerDictionary.TryGetValue(fileHandle,out var filePointer))
+            {
```
Comment (enusbaum) [inline MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs:7717]: Before the `if`, you can declare `var result = -1L` and if the value exists, set `result = filePointer.Length` (no need to cast as `result` will already be a `long`).

This way you can just have a single statement setting the registers to the value of `result`. Reduces complexity of the method 👍

File: MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs [hunk 2]
```diff
@@ -14,6 +14,7 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
+using System.ComponentModel.Design;
```
Comment (enusbaum) [inline MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs:17]: Can probably remove this

**Review by enusbaum — CHANGES_REQUESTED** [review]
Great Test! Minor changes on the Ordinal

File: MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs (hunk 1 above)
Comment (tuday2) [reply MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs:7717]: Nice ok, that made is a lot simpler 👍

**Review by enusbaum — APPROVED** [review]

**Pull request merged**